src/api/python/hidra/utils/utils_netgroup.py

```diff
@@ -0,0 +1,31 @@
+
+from ctypes import *
+
+def find_hostname(netgroup):
+
+    libc = CDLL("libc.so.6")
+    if (libc != None):
+        pass
+    else:
+        libc = CDLL("libc.so.7")
+
+ 
+    hosts = None
+    if libc.setnetgrent(netgroup):
```

Comment:
One needs to check that `netgroup` is of the correct type, otherwise this might cause a segfault.